.github/workflows/tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         run: deno lint
 
       - name: Run tests
-        run: deno task test
+        run: deno run test
```

Comment:
FYI: Changing this is more personal preference. The `task` command will be the same in Deno 2 as it is today.

Reply:
Honesty, I don't like this change in Deno because it opens the door to inconsitencies and makes the API even more complicated and confused than it's currently:

- `deno run` can run files and tasks
- `deno ...` only can run files but not tasks
- what does `deno run npm:foo` run? the `npm:foo` package or the `npm:foo` task?
- I have been eagerly waiting for this https://github.com/denoland/deno/issues/19955 in order to simplify the `lume` task, but now, [if it's merged](https://github.com/denoland/deno/pull/20018) it will create more inconsistencies. For example, what does `deno fmt` run? the `fmt` bare specifier from import maps (if exists) or the `fmt` command of Deno?

`deno task ...` and `deno run ...` were clear and easy to understand. You don't have to do ALL what Node/NPM do.